### DISCUSSION TITLE
Document symbols shows a lot of empty lines in neovim kickstart

### DISCUSSION THREAD

**Discussion by ziriuz84** [folke/trouble.nvim#603] [Q&A]
I'm configuring Trouble in my new installation of Neovim via kickstart.nvim. I use default options for Trouble. When I launch document symbols in a php file it appears as the image 

![image](https://github.com/user-attachments/assets/edfeb78a-caab-49a2-bafc-37d7628c41e0)

How can I know what are all those empty lines and how can I remove them if unuseful?

**Comment by mfandrade** [comment]
Same here.

**Comment by bmatcuk** [comment]
I ran into this problem, too. It's caused by the default option `multiline = true`. They look like a bunch of empty lines, but that's only because the text is off the right side of the screen. If you open the symbols with `:Trouble lsp_document_symbols`, which will open at the bottom of your screen by default, you'll see what I mean.

You can fix the problem a couple different ways. You could, of course, disable `multiline` across the board, by setting `multiline = false` in your top-level Trouble config, but that's probably not what you want. You could create your own mode that inherits from `symbols` like:

```lua
...
modes = {
  mysymbols = {
    mode = "symbols",
    multiline = false,
  }
}
...
```

And then use it with `:Trouble mysymbols`. Or, you could copy/paste the [symbols mode](https://github.com/folke/trouble.nvim/blob/f176232e7759c4f8abd923c21e3e5a5c76cd6837/lua/trouble/config/init.lua#L132-L160) from the default configuration into your config, and add `multiline = false` somewhere in there.
